client/table.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Peter Mattis (peter@cockroachlabs.com)
+
+package client
+
+import (
+	"log"
+
+	"github.com/cockroachdb/cockroach/client"
+	"github.com/cockroachdb/cockroach/proto"
+)
+
+// CreateTable ...
+func (db *DB) CreateTable(name string, schema proto.TableSchema) error {
+	return nil
+}
+
+// RowBuilder ...
+type RowBuilder struct {
+	tableName string
+	columns   []string
```

Comment:
That's what I was thinking (some sugar might be in order to avoid typing out the whole map if it's mostly nil, maybe `.WantRows("a", "b")`):

```
r.Rows({"id": 9}).WantRows("name", "address")
```